stginga/plugins/local_plugin_mixin.py

```diff
@@ -26,20 +26,27 @@
 class HelpMixin(object):
     def help(self):
         """Display online help for the plugin."""
-        if not self.fv.gpmon.has_plugin('WBrowser'):
-            self._help_docstring()
-            return
+        if self.fv.gpmon.has_plugin('WBrowser') and Widgets.has_webkit:
```

Comment:
My clean installation didn't pick up webkit. I cannot find any `python-webkit` on PyPI. How do I install it?

Reply:
In `conda` it can be installed by `pyqtwebengine`. It looks like PyPI has `PyQtWebEngine`, but I would install it via the same package manager as you installed pyqt otherwise you are likely to run into problems. This is one reason I want to get rid of it--it doesn't seem to be installed by default when one installs pyqt.